src/components/Navigation.astro

```diff
@@ -64,7 +64,7 @@ const navItems = [
       id="navbar-default"
     >
       <ul
-        class="p-5 font-medium flex flex-col md:p-0 rounded-lg md:flex-row md:space-x-6 lg:space-x-20 rtl:space-x-reverse md:mt-0 bg-blackee"
+        class="space-y-2 md:space-y-0 p-5 font-medium flex flex-col md:p-0 rounded-lg md:flex-row md:space-x-6 lg:space-x-20 rtl:space-x-reverse md:mt-0 bg-blackee"
```

Comment:
`space-y-[number]` (https://tailwindcss.com/docs/space) creates a space between the elements `y` means `y-axis` or top to bottom

we want that for the mobile versions only, so  we cancel the spacing by another class `md:space-y-0` from the screen size `md` the space between the elements on `y-axis` will be 0